docs/api/websocket.md

```diff
@@ -1,9 +1,21 @@
 ---
 title: "WebSocket API"
 ---
-
 Home Assistant contains a WebSocket API. This API can be used to stream information from a Home Assistant instance to any client that implements WebSockets. We maintain a [JavaScript library](https://github.com/home-assistant/home-assistant-js-websocket) which we use in our frontend.
 
+
+## Endpoint Address
```

Comment:
This section is confusing. This information should be spread over all the different sections.

Also long lived access tokens are not the typical use case. 99% of interaction is using refresh tokens, so drop that. 

Default port number or WS integration is also not important, that's all just how HA is hosted.